rds-deployment-ha-gateway/azuredeploy.json

```diff
@@ -367,7 +283,7 @@
             "typeHandlerVersion": "1.8",
             "autoUpgradeMinorVersion": true,
             "settings": {
-              "fileUris": [ "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/rds-deployment-ha-gateway/Scripts/WebAndGwFarmAdd_PostConfig1.1.ps1" ]
+              "fileUris": [ "https://raw.githubusercontent.com/Alan-AcutePath/RDS-Templates/master/rds-deployment-ha-gateway/Scripts/WebAndGwFarmAdd_PostConfig1.1.ps1" ]
```

Comment:
must point at Azure/RDS-Templates

suggestion: use assetLocation parameter, and use concat() function